db/migrate/20200803125737_add_icon_to_programming_languages.rb

```diff
@@ -0,0 +1,22 @@
+class AddIconToProgrammingLanguages < ActiveRecord::Migration[6.0]
+
+  @@icon_map = {
```

Comment:
```suggestion
  ICON_MAP = {
```
Creates a constant instead of class variable.